nurseconnect_registration/settings.py

```diff
@@ -153,3 +153,8 @@
 OPENHIM_AUTH = (OPENHIM_USERNAME, OPENHIM_PASSWORD)
 
 CELERY_BROKER_URL = env("CELERY_BROKER_URL", str, "amqp://")
+
+JEMBI_URL = env("JEMBI_URL", str, "REPLACEME")
+JEMBI_USERNAME = env("JEMBI_USERNAME", str, "REPLACEME")
+JEMBI_PASSWORD = env("JEMBI_PASSWORD", str, "REPLACEME")
+JEMBI_AUTH = (JEMBI_USERNAME, JEMBI_PASSWORD)
```

Comment:
We can probably use the `OPENHIM_...` settings I defined above, to avoid duplicating the settings with two different names.